src/components/standalone/openvpn_rw/UserConnectionsTable.vue

```diff
@@ -0,0 +1,212 @@
+<!--
+  Copyright (C) 2024 Nethesis S.r.l.
+  SPDX-License-Identifier: GPL-3.0-or-later
+-->
+
+<script setup lang="ts">
+import { useI18n } from 'vue-i18n'
+import {
+  NeTable,
+  NeTableHead,
+  NeTableHeadCell,
+  NeTableBody,
+  NeTableRow,
+  NeTableCell,
+  NePaginator,
+  useItemPagination,
+  formatDateLoc,
+  byteFormat1024,
+  useSort,
+  NeSortDropdown
+} from '@nethesis/vue-components'
+import type { ConnectionsRecord } from './ConnectionsHistory.vue'
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+import { ref, computed } from 'vue'
+const { t } = useI18n()
+
+const props = defineProps<{
+  connectionsRecords: ConnectionsRecord[]
+}>()
+
+const pageSize = ref(10)
+
+const sortKey = ref('startTime')
+const sortDescending = ref(true)
+
+function compareIpAddresses(ip1: string, ip2: string): number {
+  const octets1 = ip1.split('.').map(Number)
+  const octets2 = ip2.split('.').map(Number)
+
+  for (let i = 0; i < 4; i++) {
+    if (octets1[i] !== octets2[i]) {
+      return octets1[i] - octets2[i]
+    }
+  }
+  return 0
+}
+
+// Custom sorting functions
+const sortFunctions = {
+  account: (a: ConnectionsRecord, b: ConnectionsRecord) => {
```

Comment:
Inside `sortFunctions` we need to specify only custom sorting functions: default sorting functions are already implemented in [useSort.ts composable](https://github.com/nethesis/vue-components/blob/main/src/composables/useSort.ts#L14).

Therefore, `account` key and function can be removed, and probably also `startTime`, `endTime`, `duration`, `bytesReceived` and `bytesSent`. You can try to remove them and check if the table sorting still works.

On the other hand, `virtualIpAddress` and `remoteIpAddress` are definitely needed here.

Reply:
![image](https://github.com/user-attachments/assets/8c5e385b-4be6-44db-8450-5480cc4b63d6)
![image](https://github.com/user-attachments/assets/c851e9ca-d02c-4637-90f1-1bd1a91e947d)
if you do not mind I prefer to keep my function for endTime and duration it seems that the missing value replaced by `-` makes some strange behaviour with the default sorting function

Reply:
If the strange behavior is that rows without a value appear first or last, that's perfectly normal. Anyway non problem, you can leave it as it is now.